backend/matching/data/data_generator.py

```diff
@@ -1,7 +1,11 @@
 import argparse
 import sys
-from llsc.backend.matching.data.seeder.data_seeder import Seeder
-from config import OUTPUT_FORMAT_CHOICES, FILE_PATH_REQUIRED_FORMATS
+from backend.matching.data.seeder.data_seeder import Seeder
```

Comment:
I'd try to make this `backend` folder centric, or you can use local file paths `.seeder.data_seeder`. I can't access this from `backend` iirc